.travis.yml

```diff
@@ -10,27 +10,23 @@ env:
   global:
     - RUN_PHPSTAN="FALSE"
   matrix:
-    - LOWEST_DEPS=""
-    - LOWEST_DEPS="--prefer-lowest"
+    - PREFER_LOWEST="" WITH_COVERAGE="--coverage-clover=coverage.xml"
+    - PREFER_LOWEST="--prefer-lowest" $WITH_COVERAGE=""
 
 matrix:
   include:
   - name: 'PHPStan'
-    php: 7.2
+    php: 7.4
     env: RUN_PHPSTAN="TRUE"
-
-install:
-  - if [ $RUN_PHPSTAN == "TRUE"  ]; then composer require --dev phpstan/phpstan:^0.12; fi
+  fast_finish: true
 
 before_script:
-  - composer update --prefer-source $LOWEST_DEPS
+  - composer update --prefer-source $PREFER_LOWEST
 
 script:
-  - if [ $RUN_PHPSTAN == "FALSE"  ]; then ./bin/php-cs-fixer  fix --dry-run --diff; fi
-  - if [ $RUN_PHPSTAN == "FALSE"  ]; then ./bin/phpunit --configuration tests/phpunit.xml --coverage-clover=coverage.xml; fi
-  - if [ $RUN_PHPSTAN == "TRUE"  ]; then ./bin/phpstan.phar analyse -c phpstan.neon lib; fi
+  - if [ $RUN_PHPSTAN == "FALSE" ]; then php vendor/bin/php-cs-fixer fix --dry-run --diff; fi
+  - if [ $RUN_PHPSTAN == "FALSE" ]; then php vendor/bin/phpunit --configuration tests/phpunit.xml $WITH_COVERAGE; fi
+  - if [ $RUN_PHPSTAN == "TRUE" ]; then php vendor/bin/phpstan analyse -c phpstan.neon lib tests; fi
```

Comment:
in case we decide it on the other PR: this paths could be part of the phpstan.neon config-file

Reply:
yes, if we decide to change the "template" `phpstan.neon` then we can make another PR to each repo.